src/simulate/config/config.yml

```diff
@@ -0,0 +1,47 @@
+http:
+  url: http://lti.tools/caliper/event?key=udp_dev
+  token: uy76fuo0gDRT
+users:
+    - id  : user001
+      user_sis_id : user001_sisid
+      user_canvas_id : user001_cid
+    - id  : user002
+      user_sis_id : user0012_sisid
+      user_canvas_id : user002_cid
+    - id  : user003
+      user_sis_id : user003_sisid
+      user_canvas_id : user003_cid
+    - id  : user004
+      user_sis_id : user004_sisid
+      user_canvas_id : user004_cid
+    - id  : user005
+      user_sis_id : user005_sisid
+      user_canvas_id : user005_cid
+courses:
+    - course_site : https://leccap.engin.umich.edu/leccap/site/iosb1eezb220x49khge
+      course_name : BIOLOGY 172
+      course_sis_id : 216004646201001
+      course_canvas_id: 182678
+      video_player:
+          - id : https://leccap.engin.umich.edu/leccap/viewer/r/woNYn8
+            player_name : molecules study
+          - id : https://leccap.engin.umich.edu/leccap/viewer/r/FeLqjA
+            player_name: Immunology
```

Comment:
now the `object['name']` is pulled from the config.yml and not randomized as i was doing it earlier

Reply:
The name is in this YAML file, but you're still generating a random name for the video:

```python
             object['name'] = 'Lecture recorded on ' + utils.get_current_date()
 ```

From [line 59 of src/simulate/event_transformer/Transformer.py](https://github.com/tl-its-umich-edu/caliper-event-analysis/pull/14/commits/11e53b3af07319a9240f708bb62bdf5192f4dd67#diff-c08086034f96dff9a46b424fda586e9aR59).